### PR TITLE
List items and some changes in js

### PR DESCRIPTION
**Pull request by Piotr-code-tech**

### CODE REVIEW

File: src/calculator.js [hunk 1]
```diff
@@ -47,6 +93,21 @@ const resetEvent = () => {
     reset();
 }
 
+const openWindowEvent = () => {
```
Comment (pstudniak) [inline src/calculator.js:96]: this event functions are redundant

File: src/calculator.js [hunk 2]
```diff
@@ -47,6 +93,21 @@ const resetEvent = () => {
...
+    openWindow();
+}
+
+const closeWindowEvent = () => {
```
Comment (pstudniak) [inline src/calculator.js:100]: also apart from redundancy, those are not Events, those are Callbacks to Events

File: src/index.html [hunk 3]
```diff
@@ -1,52 +1,80 @@
...
-                <p class="valueInputHead">Please enter the value :</p>
+                <p class = "valueInputHead">Please enter the value :</p>
             </div>
-            <div>
-                <input type="number" name="value" id="writtenValue" min="1" placeholder="Value" required>
-                <button class="calculateButton">Calculate</button>
-                <button class="resetButton">Reset</button>
+            <div class = "inputContainer">
+                <input type = "number" name = "value" id = "writtenValue" min = "1" placeholder = "Value" required>
+                <button class = "calculateButton">Calculate</button>
```
Comment (pstudniak) [inline src/index.html:20]: no need for spaces in html attributes values assignment

File: src/calculator.js [hunk 4]
```diff
@@ -34,7 +37,50 @@ const displayResult = (resultValue, writtenValue, choosedRadioButton) => {
...
+    for(const property in rowContent){
+        let newColumn = document.createElement("td");
+        newColumn.innerHTML = rowContent[property];
+        newRow.appendChild(newColumn);
+    }
+
+    let containerForButton = document.createElement("td");
+    let deleteButton = document.createElement("button");
+    deleteButton.className = "deleteRowButton";
+    deleteButton.innerHTML = "\u00D7";
```
Comment (pstudniak) [inline src/calculator.js:75]: ? is it some kind of html entity?

**Review by pstudniak — CHANGES_REQUESTED** [review]
This PR looks good overall. Now we will transist it to app with build system. We will split files logically so we could get rid of any comments from the code that are organising structure so far. 

Please apply requested changes and merge PR. 

Thanks!

File: src/calculator.js (hunk 4 above)
Comment (Piotr-code-tech) [reply src/calculator.js:75]: it is a minus icon attached to the button

File: src/index.html (hunk 3 above)
Comment (Piotr-code-tech) [reply src/index.html:20]: okey, I think that code will be more readable, but you're right it isn't the best way

**Pull request merged**